src/shogun2-security/src/test/java/de/terrestris/shogun2/security/Shogun2AuthenticationProviderTest.java

```diff
@@ -195,6 +202,31 @@ public void authenticate_shouldThrowBadCredentialsExceptionIfPasswordDoesNotMatc
 	}
 
 	/**
+	 * Tests whether a {@link DisabledException} is thrown when the user is not
+	 * active.
+	 */
+	@Test(expected=DisabledException.class)
+	public void authenticate_shouldThrowDisabledExceptionIfUserIsInactive() {
+
+		// 1. Mock an authentication request object
+		final String shogun2UserName = "user";
+		final String correctPassword = "correctPassword";
+		final User userToAuth = createUserMock(shogun2UserName, correctPassword);
+
```

Comment:
Please set the `active` flag explicitly to `false` (although it is the default value).

Possibly it makes sense to refactor the `createUserMock` method by adding an additional parameter `active`...!?